deploy/base/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
         - name: ai-config
           configMap:
             name: ai-config
+        - name: model-config
```

Comment:
In this case, we are associating the word "model" with just the "embeddings". That can be a bit confusing. There are many other "models" that are or could be used. For example, we already have the Named-Entity Recognition "model" for place recognition. So to avoid confusion, I propose we take EITHER of the two suggestions below:

1. Keep this called "model-config" but update the "file" in model.yaml to be called "embeddings_file".
2. Update "model-config" and "model.yaml" to be called "embeddings-config" and "embeddings.yaml".

I think (1) makes most sense because then we can put any/all model related settings/constants in that file and it will be understood with embeddings are just one type of model.

Reply:
Very nice suggestions, updated based on 1